en/developer/developer-broker-stream-connector.md

```diff
@@ -293,62 +293,66 @@ get hostnames, etc...
     instance name corresponding to the instance id.
-9.  ``get_metric_mapping(metric_id)`` that gets from the cache the
+9.  `get_metric_mapping(metric_id)` that gets from the cache the
     metric mapping object of the given metric id. The result is a table
-    containing two keys, ``metric_id`` and ``index_id``.
-10. ``get_service_description(host_id,service_id)`` that gets from the cache the
+    containing two keys, `metric_id` and `index_id`.
+10. `get_service_description(host_id,service_id)` (**API v1 only**) that gets from the cache the
     service description of the given pair host_id / service_id. This function
     returns a string or *nil* otherwise.
-11. ``get_servicegroup_name(id)`` that gets from the cache the service group name*
+11. `get_servicegroup_name(id)` that gets from the cache the service group name*
     of the given id. This function returns a string or *nil* otherwise.
-12. ``get_servicegroups(host_id, service_id)`` that gets the list of service
+12. `get_servicegroups(host_id, service_id)` that gets the list of service
     groups containing the service corresponding to the pair *host_id* /
     *service_id*. The return value is an array of objects, each one containing
     two fields, *group_id* and *group_name*.
-13. ``get_notes(host_id[,service_id])`` that gets the notes configured in the
+13. `get_notes(host_id[,service_id])` that gets the notes configured in the
     host or service. The *service_id* is optional, if given we want notes from
     a service, otherwise we want notes from a host. If the object is not found
     in cache, *nil* is returned.
-14. ``get_notes_url(host_id[, service_id])`` that gets the notes url configured
+14. `get_notes_url(host_id[, service_id])` that gets the notes url configured
     in the host or service. The *service_id* is optional, if given we want
     *notes url* from a service, otherwise we want it from a host. If the object
     is not found in cache, *nil* is returned.
-15. ``get_action_url(host_id)`` that gets the action url configured in the host
+15. `get_action_url(host_id)` that gets the action url configured in the host
     or service. The *service_id* is optional, if given we want *action url* from
     a service, otherwise we want it from a host. If the object is not found in
     cache, *nil* is returned.
-16. ``get_severity(host_id[,service_id])`` that gets the severity of a host or
+16. `get_severity(host_id[,service_id])` that gets the severity of a host or
     a service. If you only provide the *host_id*, we suppose you want to get
     a host severity. If a host or a service does not have any severity, the
     function returns a *nil* value.
+17. `get_host(host_id)` (**API v2 only**) that returns an associative array
+    proviing all the available information for one host from the cache.
+18. `get_service(host_id, service_id)` (**API v2 only**) that returns an associative array
+    proviing all the available information for one service from the cache.
```

Comment:
In fact, we recommend these functions to be used with API v2, but they both work with API v1.